packages/babel-plugin-transform-runtime/src/index.js

```diff
@@ -1,4 +1,5 @@
 import definitions from "./definitions";
+import getHelper from "babel-helpers";
```

Comment:
Hmm, somewhat hesitant to have this as a dependency since it means we now have two parallel sources of helpers.